Generate SBOMs step failing in dotnet-docker-tools-eng-validation pipeline

```
Starting: Generate SBOMs
==============================================================================
Task         : PowerShell
Description  : Run a PowerShell script on Linux, macOS, or Windows
Version      : 2.237.5
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/powershell
==============================================================================
Generating script.
========================== Starting Command Output ===========================
/usr/bin/pwsh -NoLogo -NoProfile -NonInteractive -Command . '/mnt/vss/_work/_temp/c4d0b84b-ac51-4c7d-a7ec-47d4473b4363.ps1'
InvalidOperation: /mnt/vss/_work/_temp/c4d0b84b-ac51-4c7d-a7ec-47d4473b4363.ps1:7
Line |
   7 |  … taskDir = $(Get-ChildItem -Recurse -Directory -Filter "ManifestGenera …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot index into a null array.

##[error]PowerShell exited with code '1'.
Finishing: Generate SBOMs
```

This is a regression from the changes in https://github.com/dotnet/docker-tools/pull/1283. That presumed that `ManifestGeneratorTask` is being automatically injected and called by the 1ES templates, via the `Generate SBoM Manifest` step. That's true for the logic in the product repos but not for the validation pipeline in this repo which doesn't include that injected step.

--- a/eng/common/templates/1es-unofficial.yml
+++ b/eng/common/templates/1es-unofficial.yml
@@ -44,6 +44,8 @@ extends:
         ignoreDirectories: $(Build.SourcesDirectory)/versions
         whatIf: true
         showAlertLink: true
+      sbom:
+        enabled: ture
       sourceRepositoriesToScan:
         exclude:
         - repository: InternalVersionsRepo

--- a/eng/common/templates/1es-unofficial.yml
+++ b/eng/common/templates/1es-unofficial.yml
@@ -45,7 +45,7 @@ extends:
         whatIf: true
         showAlertLink: true
       sbom:
-        enabled: ture
+        enabled: true
       sourceRepositoriesToScan:
         exclude:
         - repository: InternalVersionsRepo

